.github/workflows/alex_reviewdog.yml

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Should this be changed to 2021 now? @cdeshmukh

Reply:
Yes

Reply:
Done